lib/kitchen/oven.rb

```diff
@@ -12,16 +12,10 @@ class Oven
     # @param recipes [Array<Recipe>] an array of recipes with which to bake the document
     # @param output_file [String] the path to the output file
     #
-    def self.bake(input_file:, recipes:, output_file:, config_file: nil, locales: nil)
+    def self.bake(input_file:, recipes:, output_file:, config_file: nil)
       profile = BakeProfile.new
       profile.started!
 
-      if locales
-        I18n.load_path << file_glob("#{__dir__}/#{locales}/*.yml")
```

Comment:
I found that load_path stuff was maybe too heavy-handed, as additions to it stick around.  When I looked into it more, I decided to go for a solution where a recipe's translations were added temporarily.  Also the solution in my PR doesn't change the Kitchen backend, it instead makes a new recipe-specific backend and temporarily chains it in front of the standard Kitchen I18n backend.